packages/storybook8/stories/Components/MessageThread/snippets/placeholdermessages.ts

```diff
@@ -85,6 +85,114 @@ export const GetHistoryWithCustomMessages = (): (CustomMessage | ChatMessage)[]
   ];
 };
 
+export const GetHistoryWithAdaptiveCardMessages = (adaptiveCardTitle?: string): (CustomMessage | ChatMessage)[] => {
+  // Adaptive Cards samples and templates can be found in https://adaptivecards.io/samples/
+  const adaptiveCard = {
+    type: 'AdaptiveCard',
+    body: [
+      {
+        type: 'TextBlock',
+        size: 'medium',
+        weight: 'bolder',
+        text: adaptiveCardTitle || 'Publish Adaptive Card Schema',
+        style: 'heading',
+        wrap: true
+      },
+      {
+        type: 'ColumnSet',
+        columns: [
+          {
+            type: 'Column',
+            items: [
+              {
+                type: 'Image',
+                style: 'person',
+                url: 'https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg',
```

Comment:
Will this image always exist?

Reply:
@palatter These data is token from the adaptivecards website https://adaptivecards.io/samples/.
Hopefully it will be well maintained.